### PR TITLE
fix: 🐛 made autocomplete component fully disabled when disabled 

### PR DESCRIPTION
**Pull request by Chris-Boyle**
✅ Closes: #135

Users could still clear the value and access the drop down to select other values when they passed in the `isDisabled` prop to the SQFormAutoComplete component.  I have removed those two pieces of functionality when `isDisabled` is true.

freeSolo docs(not rock climbing): https://material-ui.com/components/autocomplete/#free-solo
disabledClearable : https://material-ui.com/components/autocomplete/#playground

Loom: https://www.loom.com/share/c69a8a7e0c6943129bfc55e520b0bcd2
Fully disabled
image

Story example
image
image

### CODE REVIEW

File: stories/SQForm.stories.js [hunk 1]
```diff
@@ -526,6 +526,7 @@ export const applyAnAction = () => {
           label="Actions with a default value"
           size={5}
           isRequired={true}
+          isDisabled
```
Comment (SeanGroff) [inline stories/SQForm.stories.js:529]: @Chris-Boyle Could you make this boolean value a knob in storybook the user can toggle enable/disabled state, please?

Comment (Chris-Boyle) [reply stories/SQForm.stories.js:529]: Yes!

File: src/components/SQForm/SQFormAutocomplete.js [hunk 2]
```diff
@@ -164,6 +164,8 @@ function SQFormAutocomplete({
         onInputChange={handleInputChange}
         inputValue={inputValue}
         value={initialValue}
+        disableClearable={isDisabled}
+        freeSolo={isDisabled}
```
Comment (SeanGroff) [inline src/components/SQForm/SQFormAutocomplete.js:168]: `freeSolo` sounds like a 2000's emo band lol

**Review by SeanGroff — APPROVED** [review]
Great discovery thanks for fixing this @Chris-Boyle 💯

**Comment by 20BBrown14** [comment]
Is this issue present on the async version as well?

**Comment by Chris-Boyle** [comment]
@20BBrown14 I am not familiar with that one.  If it is then these changes probably just need to be copied over.  We can log another issue if it is.

**Pull request merged**